Public/Src/App/Bxl/Args.cs

```diff
@@ -203,6 +203,9 @@ public bool TryParse(string[] args, PathTable pathTable, out ICommandLineConfigu
                             "additionalConfigFile",
                             "ac",
                             opt => ParsePathOption(opt, pathTable, startupConfiguration.AdditionalConfigFiles)),
+                        OptionHandlerFactory.CreateOption(

```

Comment:
For command line switches that make sense for macOS only we currently use `#if PLATFORM_OSX`.

That said, maybe it would make more sense to use `#if !PLATFORM_OSX` here instead of checking and throwing below (line 1224)

Reply:
This may potentially be used in `PLATFORM_OSX`. For example, we may want to run pip with super user privilege. Currently, the execution mode name comes under the same umbrella called 'admin'. 

For the time being, we'll keep the option as is without enclosing it in `#if !PLATFORM_OSX`.

---
In reply to: [277332947](https://github.com/Microsoft/BuildXL/pull/159#discussion_r277332947) [](ancestors = 277332947)